packages/config/config/devices/0x0098/ct101.json

```diff
@@ -52,29 +52,40 @@
 					"value": 0
 				},
 				{
-					"label": "0.5°",
+					"label": "0.5° F",
 					"value": 1
 				},
 				{
-					"label": "1.0°",
+					"label": "1.0° F",
 					"value": 2
 				},
 				{
-					"label": "1.5°",
+					"label": "1.5° F",
 					"value": 3
 				},
 				{
-					"label": "2.0°",
+					"label": "2.0° F",
 					"value": 4
 				}
 			]
 		},
+		"2": {
```

Comment:
Please check the manul and list out these options. If there are only four they should be presented as options through the UIs.

Reply:
This param is a read-only byte array. I'm not quite sure how to represent this using options, as some of them are dynamic based on the number or stages:
```
2. HVAC Settings (32-bit)
Byte 1 = HVAC Setup: Normal (0x01) or Heat Pump (0x02)
Byte 2 = Aux Setup(Gas (0x01) or Electric (0x02)) & Number of Auxiliary Stages (Heat Pump)/Number of Heat Stages (Normal)
Byte 3 = Number of Heat Pump Stages
Byte 4 = Number of Cool Stages
```